File: server/Models/Res/UserRes.cs
namespace connecto.server.Models.Res;

public class UserRes
{
    public Guid Id { get; set; }

    public string? FirstName { get; set; }

    public string? LastName { get; set; }

    public string Username { get; set; } = null!;

    public DateTimeOffset CreatedAt { get; set; }
}

File: clients/java/src/main/java/com/connecto/client/models/UserRes.java
package com.connecto.client.models;

import com.google.gson.annotations.SerializedName;
import java.time.OffsetDateTime;
import java.util.UUID;

public class UserRes {
    public UUID id;
    public String firstName;
    public String lastName;
    public String username;
    @SerializedName("createdAt")
    public OffsetDateTime createdAt;
}
